scalecube-configuration/src/test/java/io/scalecube/configuration/it/ReadListIT.java

```diff
@@ -9,4 +10,15 @@
 
 @ExtendWith(Fixtures.class)
 @WithFixture(value = IntegrationEnvironmentFixture.class, lifecycle = Lifecycle.PER_METHOD)
-final class ReadListIT extends ReadListScenario {}
+final class ReadListIT extends ReadListScenario {
+
+  @Override
+  protected void readEntries(ConfigurationService configurationService) {
```

Comment:
Why did you override it?

Reply:
It's to long to test it should be to increase waiting for update from 1 to few seconds in each method and to create new env. So IT is going not a minute but 5-7 minutes. I'll explain you at office